doc/api/next_api_changes/deprecations/18680-AL.rst

```diff
@@ -0,0 +1,8 @@
+wx backend cleanups
+~~~~~~~~~~~~~~~~~~~
+The *origin* parameter to ``_FigureCanvasWxBase.gui_repaint`` is deprecated
+with no replacement; ``gui_repaint`` now automatically detects the case where
+it is used with the wx renderer.
+
+The ``NavigationToolbar2Wx.get_canvas`` method is deprecated; copy its
```

Comment:
What's the motivation to deprecate this. Getting the canvas seems like a useful action, and `type(toolbar.canvas)(frame, -1, figure)` is pretty arcane.

Reply:
That method is not present on any of the other backends NavigationToolbar classes, and I'd rather try to make the NavToolbar API consistent across classes.

Reply:
I would leave out `type` here, and use the explicit class instead.